### DISCUSSION TITLE
Why do S3 Hooks not have the method to list versions

### DISCUSSION THREAD

**Discussion by dev-manojm** [apache/airflow#27653] [Ideas]
I have been using airflow since quite a long now. And I want to know why is there no provision for List Versions method in [S3Hooks](https://github.com/apache/airflow/blob/main/airflow/providers/amazon/aws/hooks/s3.py). I know that List Versions is available in AWSHook but why not in S3 Hooks? Can i add it in S3 Hooks, If yes whats the procedure?

**Comment by potiuk** [comment]
It's not there because no-one added it. As simple as that.

Sure - Airflow is created by > 2200 users and you can become one of the contributors. Everything needd to learn how to contribute and what to do is descibed in https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst - including "quick starts", how to set-up environment, how to communicate, what to expect from reviews etc. etc. 

It's very comprehensive and it allowed more than 2000 users to become contributors - I hope we have one more contributor in the making here.
